### ISSUE TITLE
Compiling mofs on linux

### ISSUE THREAD

**Issue by mrhockeymonkey** [PowerShell/PowerShell#10183]
In the latest release of powershell 7 preview 2 one of the features listed is support for compiling mofs on linux (#9834). I took this to mean that I could use a linux box to compile mofs that will later be applied to windows machines...

I spent the best part of today trying to achieve just that but sadly with little success. I am using CentOS 7.6 and have installed [omi](https://github.com/microsoft/omi/releases) and [dsc-for-linx](https://github.com/microsoft/PowerShell-DSC-for-Linux/releases) but after having done so Get-DscResource does not return any resources

I also tried running a configuration in case it was just a bug with the output but I get the below (CertificateDSC was just one of many stabs in the dark):
```powershell
At line:6 char:9
+         CertificateImport MyTrustedRoot
+         ~~~~~~~~~~~~~~~~~
Undefined DSC resource 'CertificateImport'. Use Import-DSCResource to import the resource.
+ CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
+ FullyQualifiedErrorId : ResourceNotDefined
```

I know this is a work in progress but will this eventually be a possibility? If so are there any issues/milestones to watch progress towards this goal? Or is it that only the linux specific "nx" resources will ever be available on a linux host?

**Comment by mrhockeymonkey** [comment]
Would anyone be able to shed some light on this? I thought i'd come back and try again now that preview 6 is released and no more features will be added. This time I tried Ubuntu Xenial, installed omi, dsc and pwsh-preview but I get the same issue. No DSC Resources are found. 

These are the docs for reference: https://docs.microsoft.com/en-us/powershell/scripting/dsc/getting-started/lnxgettingstarted?view=powershell-6

**Comment by mrhockeymonkey** [comment]
It looks like this was fixed in 7.0.0-rc.2. Compiling mofs now works fine. Sadly compiling with credentials doesn't work but i've opened #11656 as it is a separate issue and I will close this one.
